app/bundles/CampaignBundle/EventListener/CampaignInjectCustomContentSubscriber.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\CampaignBundle\EventListener;
+
+use Mautic\CampaignBundle\Entity\Campaign;
+use Mautic\CoreBundle\CoreEvents;
+use Mautic\CoreBundle\Event\CustomContentEvent;
+use Mautic\CoreBundle\Model\AuditLogModel;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Twig\Environment;
+
+class CampaignInjectCustomContentSubscriber implements EventSubscriberInterface
```

Comment:
New PHP classes should be final by default. Not final only if there is a reason for it.
```suggestion
final class CampaignInjectCustomContentSubscriber implements EventSubscriberInterface
```